src/tango/quic/fd_quic.c

```diff
@@ -127,6 +133,13 @@ fd_quic_footprint_ext( fd_quic_limits_t const * limits,
   if( FD_UNLIKELY( !tls_footprint ) ) { FD_LOG_WARNING(( "invalid fd_quic_tls_footprint" )); return 0UL; }
   offs                += tls_footprint;
 
+  /* allocate space for fd_ip_t */
```

Comment:
How did this come back? I moved it to be passed in by the caller, it makes the seccomp logic cleaner.